pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -62,10 +64,14 @@ function docker_network_config() {
 
 DOCKER_NETWORK_OPTIONS='${DOCKER_NETWORK_OPTIONS}'
 EOF
-
-		systemctl daemon-reload
-		systemctl restart docker.service
-
+		## linux bridge
+		ip link set lbr0 down || true
```

Comment:
Why did you have to move this here? (And not remove it from setup(); this means lbr0 will be created, then destroyed, then created again.)

Note that the comment "disable iptables for lbr0" just below this section is inaccurate; it actually disables iptables for all bridges, so it ought to work whether or not lbr0 exists yet. (If it doesn't then maybe it's because we need to modprobe something else?)